meinberlin/apps/dev/management/commands/comments.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from adhocracy4.comments.models import Comment
+from meinberlin.apps.ideas.models import Idea
+
+
+class Command(BaseCommand):
+    help = """
+    Creates fake comments for testing.
+
+    The command on its own counts fake objects currently in the database and prints a summary.
+
+    Usage:
+
+        $ ./manage.py comments --user <pk> --count <int>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            default=0,
```

Comment:
are there users with pk 0?

Reply:
oh, I added this by mistake to the PR, sorry :/ i used this to test some comment related stuff

Reply:
good to have though. could you make a separate PR and keep it?

Reply:
will do